torchgeo/datasets/benin_cashews.py

```diff
@@ -279,17 +283,20 @@ def _validate_bands(self, bands: Tuple[str, ...]) -> None:
                 raise ValueError(f"'{band}' is an invalid band name.")
 
     @lru_cache(maxsize=128)
-    def _load_all_imagery(self, bands: Tuple[str, ...] = ALL_BANDS) -> Tensor:
+    def _load_all_imagery(self, bands: Tuple[str, ...] = ALL_BANDS
+                          ) -> Tuple[Tensor, rasterio.Affine, CRS]:
```

Comment:
I'm not sure if black is going to like this. Just run `black .` to solve most formatting issues.